docs/styles.css

```diff
@@ -4,850 +4,458 @@ Fonts
-    white-space: pre-wrap;
+.grammar-syntax {
+  color: var(--pink);
 }
-
-.error pre a {
-    color: #29abe2;
+.grammar-comment {
+  color: var(--silver);
 }
-
-.cursor {
-    animation: blinker 1s linear infinite;
+.grammar-token {
+  color: var(--green);
+}
+
+/* Codelab */
+.codelab-category-beginner {
+  border-color: var(--turquoise);
+}
+.codelab-category-beginner h2 {
+  color: var(--turquoise);
+}
+.codelab-category-intermediate {
+  border-color: var(--yellow);
+}
+.codelab-category-intermediate h2 {
+  color: var(--yellow);
+}
+.codelab-category-advanced {
+  border-color: var(--red);
+}
+.codelab-category-advanced h2 {
+  color: var(--red);
+}
+
+/* Skip to content */
+.skip-to-content {
+  border: 0;
+  clip: rect(0 0 0 0);
+  height: 1px;
+  width: 1px;
+  margin: -1px;
+  padding: 0;
+  overflow: hidden;
+  position: absolute;
+}
+.skip-to-content:focus {
+  background-color: var(--dark-gray);
+  border: 0.125rem solid var(--violet);
+  border-radius: 0.25rem;
+  color: var(--white);
+  outline: none;
+  padding: 0.5rem;
+  position: fixed;
+  left: 1rem;
+  width: auto;
+  height: auto;
+  clip: auto;
+  z-index: 1;
+}
+
+/* Tabs */
+.tabs__tablist {
+  background-color: var(--dark-gray);
+  display: flex;
+  margin: 0;
+  overflow-x: auto;
+  border-top-left-radius: 0.5rem;
+  border-top-right-radius: 0.5rem;
+}
+.tabs__tab {
+  border-bottom: 1px solid var(--turquoise);
+  border-top-left-radius: 0.5rem;
+  border-top-right-radius: 0.5rem;
+  cursor: pointer;
+  padding-bottom: 0.5rem;
+  padding-left: 2rem;
+  padding-right: 2rem;
+  padding-top: 0.5rem;
+  z-index: 1;
+}
+.tabs__tab--selected {
+  border-bottom: 1px solid var(--raisin-black);
+  border-left: 1px solid var(--turquoise);
+  border-right: 1px solid var(--turquoise);
+  border-top: 1px solid var(--turquoise);
+  color: var(--white);
+  background-color: var(--raisin-black);
+  font-weight: 700;
+}
+.tabs__tab:focus {
+  color: var(--turquoise);
+  font-weight: 700;
+  outline: 0;
+}
+.tabs__panels {
+  margin-top: -1px;
+}
+.tabs__panel {
+  display: none;
+}
+.tabs__panel--selected {
+  border: 1px solid var(--turquoise);
+  display: block;
+  overflow-x: auto;
+}
+
+/* Table */
+.table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+.table th {
+  border-bottom: 1px solid var(--white);
+  color: var(--turquoise);
+  min-width: 8rem;
+  padding: 0.25rem;
+  text-align: left;
+  text-transform: uppercase;
+}
+.table td {
+  border-bottom: 1px solid var(--gray);
+  min-width: 8rem;
+  padding: 0.25rem;
+}
+
+/* Blinker */
+.blinker {
+  animation: blinker 1s linear infinite;
 }
-
 @keyframes blinker {
-    49% {
-        background-color: transparent;
-    }
-    50% {
-        background-color: white;
-    }
-    100% {
-        background-color: white;
-    }
-}
\ No newline at end of file
+  49% {
+    background-color: transparent;
+  }
+  50% {
+    background-color: white;
+  }
+  100% {
+    background-color: white;
+  }
+}
+
+/* Inline code */
```

Comment:
I can't find any usages for .code-signature or .code-container. Am I missing something?

Reply:
- `code-signature` is used in `docs/lexicon.html` (compressed by default) and ` docs/lexicon_entry.html`
- `code-container` is used in several other files as `<pre class="code-container">`